app/lib/mappers/azOpeningTimesMapper.js

```diff
@@ -0,0 +1,42 @@
+const moment = require('moment');
+
+function getTimesFromString(timesString) {
+  const splitTimes = timesString.split('-');
+  const opens = splitTimes[0];
+  const closes = splitTimes[1];
+  return { closes, opens };
+}
+
+function getOpeningTimes(asOpeningTimes) {
+  const weekdays = moment.weekdays();
+  const openingTimes = {
+    alterations: {},
+    general: {},
+  };
+
+  // Get opening times for each day of week
+  weekdays.forEach((weekday) => {
+    const sessions = asOpeningTimes
+      .filter(ot => ot.OpeningTimeType === 'General' && ot.Weekday === weekday)
+      .map(wot => getTimesFromString(wot.Times));
+    openingTimes.general[weekday.toLowerCase()] = sessions;
+  });
+
+  // Get opening times for alterations/additional days
+  asOpeningTimes
```

Comment:
Could this be extracted into a function so the comment can be removed and the intent be maintained within the function name?